src/routes/group.js

```diff
@@ -1,11 +1,45 @@
+/**
+ * @swagger
+ * tags:
+ *    name: Group
+ *    description: API to manage groups.
+ */
+
 import express from 'express';
 import Group from '../controllers/group';
 
 const groups = express.Router();
 
-// TODO: Add Swagger Docs
-
-// add a new group
+/**
+ * @swagger
+ * path:
+ * /groups/:
+ *    post:
+ *      description: Creates a new group
+ *      summary: Creates a group
+ *      tags:
+ *        - Group
+ *
+ *      requestBody:
+ *        description:
+ *        required: true
+ *
+ *        content:
+ *          application/json:
+ *            schema:
+ *              type: object
+ *              properties:
+ *
+ *      produces:
+ *        - application/json
+ *      responses:
+ *        200:
+ *          description: OK
+ *        404:
+ *          description: Username/password combination is incorrect
```

Comment:
There is no Username/Password combination for group registration.

Reply:
It was just copied from user doc as a reference, because groups documentation requires some more work.